### PR TITLE
additions and corrections to NL band plan

### PR DESCRIPTION
**Pull request by daviderud**
additions and corrections to NL band plan

### CODE REVIEW

File: root/res/bandplans/netherlands.json [hunk 1]
```diff
@@ -3172,6 +3184,366 @@
...
+            "end": 141000000000
+        },
+        {
+            "name": "Radio Ham 1 mm band",
+            "type": "amateur",
+            "start": 241000000000,
+            "end": 250000000000
         }
-    ]
+	]
```
Comment (AlexandreRouma) [inline root/res/bandplans/netherlands.json:3548]: replace the tab with 4 spaces

File: root/res/bandplans/netherlands.json [hunk 2]
```diff
@@ -3122,8 +3134,8 @@
         {
             "start": 2412100000,
             "end": 2450000000,
-            "type": "utility",
-            "name": "WiFi shared with Broadband Modes 13cm ISM  |13cm Ham Band"
+            "type": "amateur",
+            "name": "WiFi shared with Broadband Modes 13cm ISM, 13cm Ham Band|"
```
Comment (AlexandreRouma) [inline root/res/bandplans/netherlands.json:3138]: there shouldn't be a pipe character at the end of the name

File: root/res/bandplans/netherlands.json [hunk 3]
```diff
@@ -743,20 +761,32 @@
...
+            "name": "60m HAM Alle modes (USB)"
         },
         {
-            "start": 5357000,
-            "end": 5359900,
+            "start": 5366000,
+            "end": 5366500,
             "type": "amateur",
-            "name": "Ch. 3 (60m) (Not NL)"
+            "name": "60m HAM Weak NB modes |"
```
Comment (AlexandreRouma) [inline root/res/bandplans/netherlands.json:786]: remove pip character at the end

File: root/res/bandplans/netherlands.json [hunk 4]
```diff
@@ -3047,17 +3077,11 @@
...
-            "type": "amateur",
-            "name": "|13cm Ham Band"
-        },
         {
             "start": 2320000000,
             "end": 2332500000,
-            "type": "satellite",
-            "name": "Satellite Digital Audio Radio Service (SDARS)"
+            "type": "amateur",
+            "name": "|13cm Ham Band Satellite  Digital Audio Radio Service (SDARS)"
```
[the remark this answers is not in the document]
Comment (daviderud) [reply root/res/bandplans/netherlands.json:3084]: I used pipes to mark the start and end of a band, when a band is divided in multiple sub-bands

File: root/res/bandplans/netherlands.json (hunk 3 above)
Comment (daviderud) [reply root/res/bandplans/netherlands.json]: I used pipes to mark the start and end of a band, when a band is divided in multiple sub-bands

**Pull request merged**